pages/profile/[id]/index.js

```diff
@@ -61,35 +59,29 @@ function Profile() {
     <>
       <Header />
       <div style={{ maxWidth: 1400, margin: "auto" }}>
```

Comment:
usually contianer should have maxSize: 1400 and padding left and right 20 pixel

Reply:
and if you make a common component for container and reuse it for other pages too it should be perfect

Reply:
already did that and used, but forgot to use in talent page.